hack/create-kind-podman-cluster.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+#
+# This script creates a kind cluster and configures it to use an insecure registry
+# running on the host OS.
+#
+# USAGE: sudo DEV_USER=myusername ./hack/create-kind-cluster-podman.sh [cluster_name]
```

Comment:
Is the advice about using `sudo` and `DEV_USER` left over from a previous iteration?

Reply:
Yes good catch thanks. Switched to in-script sudo so it could more easily get you a kubeconfig you could use as a non-root user.